test/tests/repository.js

```diff
@@ -362,15 +362,15 @@ describe("Repository", function() {
     return this.constRepository.statistics()
     .then(function(analysisReport) {
 
-      assert.equal(analysisReport.repositorySize.commits.count, 992);
-      assert.equal(analysisReport.repositorySize.commits.size, 265544);
-      assert.equal(analysisReport.repositorySize.trees.count, 2416);
-      assert.equal(analysisReport.repositorySize.trees.size, 1188325);
-      assert.equal(analysisReport.repositorySize.trees.entries, 32571);
-      assert.equal(analysisReport.repositorySize.blobs.count, 4149);
-      assert.equal(analysisReport.repositorySize.blobs.size, 48489622);
+      assert.equal(analysisReport.repositorySize.commits.count, 993);
```

Comment:
Oops, just realized we use the `nodegit/test` repo here when we should have used a separate repo that's frozen.

@implausible would you mind creating a `nodegit/test-frozen` repo when you have a minute?

Then I'll push up the old version of `nodegit/test` to that repo so we have a consistent repo for repo analysis and don't needlessly break tests for older commits each time we update the test repo.

Reply:
@zawata I created https://github.com/nodegit/test-frozen, go ahead and swap the const repo over to that. I also deleted the longpath branch from this PR so the numbers will change. Hopefully they're the same as they were before, but if not it's not the end of the world.